src/commands/get/servers/getservers.go

```diff
@@ -61,6 +64,7 @@ func getServersById(serverID string) error {
 	if err != nil {
 		return err
 	} else {
+		log.Info().Msgf("Retrieving server with [ID] %s", serverID)
```

Comment:
I feel like we should either add logs for all commands, or none of them... what do you think?

Reply:
Agree, add info logging in all commands